sgv2-filter.meta.js

```diff
@@ -3,10 +3,11 @@
 // @namespace   https://github.com/GarionCZ/sgv2-filter
 // @description Giveaway filter for SteamGifts v2
 // @author      Garion
-// @include     http://www.steamgifts.com/*
+// @include     http://www.steamgifts.com/giveaways*
```

Comment:
I didn't know SG doesn't forward on default. In this case however you could include "//steamgifts.com*" and exclude "//steamgifts.com/giveaway/*" and other unused pages.

On Jul 28, 2015 8:49 PM, GarionCZ <notifications@github.com> wrote:
> @@ -3,10 +3,11 @@
>  // @namespace   https://github.com/GarionCZ/sgv2-filter
>  // @description Giveaway filter for SteamGifts v2
>  // @author      Garion
> -// @include     http://www.steamgifts.com/*
> +// @include     http://www.steamgifts.com/giveaways*

I don't think this will work on the mainpage since without SG++ the main page is http://www.steamgifts.com, SG++ forwards it to http://www.steamgifts.com/giveaways to make things easier. This plugin should work without SG++ as well though.

---
Reply to this email directly or view it on GitHub:
https://github.com/GarionCZ/sgv2-filter/pull/2/files#r35684451